src/Fantomas.Tests/TestHelpers.fs

```diff
@@ -23,12 +23,7 @@ let private safeToIgnoreWarnings =
       "Identifiers containing '@' are reserved for use in F# code generation" ]
 
 let private isValidAndHasNoWarnings fileName source parsingOptions =
-    let allDefineOptions =
-        TokenParser.getOptimizedDefinesSets source
-        @ (TokenParser.getDefines source
-           |> List.map List.singleton)
-          @ [ [] ]
-        |> List.distinct
+    let allDefineOptions, _ = TokenParser.getDefines source
```

Comment:
It was sad that there was a copy of this both here and in CodeFormatterImpl!

Reply:
Indeed, although for the Benchmark it does not seem to have a noticeable impact that we now only call this once.
Unit tests do feel snappier though.